mmdet/ops/nms/nms_wrapper.py

```diff
@@ -1,18 +1,54 @@
+import sys
+
 import numpy as np
 import torch
+from torch.autograd import Function
+from torch.onnx.symbolic_opset9 import select, squeeze, unsqueeze
 
 from . import nms_ext
 
 
-def nms(dets, iou_thr, device_id=None):
+class NMSop(Function):
+
+    @staticmethod
+    def forward(ctx, bboxes, scores, iou_threshold):
+        # execute cpu or cuda nms
+        if bboxes.shape[0] == 0:
+            inds = bboxes.new_zeros(0, dtype=torch.long)
+        else:
+            inds = nms_ext.nms(
+                torch.cat([bboxes, scores[:, None]], -1), iou_threshold)
+        return inds
+
+    @staticmethod
+    def symbolic(g, bboxes, scores, iou_threshold):
+        boxes = unsqueeze(g, bboxes, 0)
+        scores = unsqueeze(g, unsqueeze(g, scores, 0), 0)
+        max_output_per_class = g.op(
+            'Constant', value_t=torch.tensor([sys.maxsize], dtype=torch.long))
+        iou_threshold = g.op(
+            'Constant',
+            value_t=torch.tensor([iou_threshold], dtype=torch.float))
+        nms_out = g.op('NonMaxSuppression', boxes, scores,
+                       max_output_per_class, iou_threshold)
+        return squeeze(
+            g,
+            select(
+                g, nms_out, 1,
+                g.op('Constant', value_t=torch.tensor([2], dtype=torch.long))),
+            1)
+
+
+def nms(bboxes, scores, iou_thr, device_id=None):
```

Comment:
We should make `nms` compatible with old code.
In all methods, `nms` is called with `dets` of shape (N, 5).